doc/source/serve/advanced-guides/multi-app-container.md

```diff
@@ -57,29 +59,31 @@ RUN sudo apt-get update && sudo apt-get install curl -y
 
 # Download the source code for the ResNet application into `resnet50_example.py`.
 RUN curl -O https://raw.githubusercontent.com/ray-project/ray/master/doc/source/serve/doc_code/resnet50_example.py
+
+ENV PYTHONPATH "${PYTHONPATH}:/home/ray"
 ```
 :::
 ::::
 
-Then, build the corresponding Docker images and push it to your choice of Docker registry. This tutorial uses `alice/whisper_image:latest` and `alice/resnet_image:latest` as placeholder names for the images, but make sure to swap out `alice` for a repo name of your choice.
+Then, build the corresponding Docker images and push it to your choice of Docker registry using `podman`. This tutorial uses `alice/whisper_image:latest` and `alice/resnet_image:latest` as placeholder names for the images, but make sure to swap out `alice` for a repo name of your choice.
 
 ::::{tab-set}
 :::{tab-item} Whisper
 ```bash
-# Build the Docker image from the Dockerfile
+# Build the Podman image from the Dockerfile using podman
```

Comment:
maybe just say build the "container image"? I am not sure if "podman" image is a common saying.